CHANGES.md

```diff
@@ -2,6 +2,28 @@
      This file is generated. Do not write release notes here.
      Notes for unreleased changes go in ./UNRELEASED.md -->
 
+## 0.8.1
```

Comment:
@konnov Are you OK with us no longer having this `[release]` suffix in the changes file?

Reply:
True. We do not need the suffix any longer.